gateway-ha/src/test/java/io/trino/gateway/ha/router/TestRoutingRulesManager.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.gateway.ha.router;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import io.trino.gateway.ha.config.RoutingRulesConfiguration;
+import io.trino.gateway.ha.domain.RoutingRules;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.NoSuchFileException;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatException;
+
+class TestRoutingRulesManager
+{
+    private RoutingRulesConfiguration routingRulesConfiguration;
+
+    private ObjectMapper yamlReader;
+
+    @BeforeEach
+    void setUp()
+    {
+        routingRulesConfiguration = new RoutingRulesConfiguration();
+        yamlReader = new ObjectMapper(new YAMLFactory());
```

Comment:
Why do need to initialize `yamlReader` every test? Please change to constant if there is no reason.

Reply:
makes sense, i'll update it.